test/mps.jl

```diff
@@ -481,6 +481,22 @@ function basicRandomMPS(N::Int; dim=4)
   return M
 end
 
+function TestCorreltationMatrix(psi::MPS,ops::Vector{Tuple{String, String}})
```

Comment:
Minor thing, but should be "Correlation" matrix.

Reply:
Also, would be nice to follow the Julia style and call it `test_correlation_matrix`.

In cases like this, I've also gone with the style of defining a function like `check_correlation_matrix` and return either `true` or `false`, then use `@test check_correlation_matrix(...)`. I think it makes it clearer it is using `@test`. An advantage of this is that it is more versatile, for example it could be used outside the context of testing (such as for a debug check in the code).